index.html

```diff
@@ -255,11 +252,36 @@ <h3 class="one">Skills</h3>
               </div>
               <img src="images/Separator.png" class="Separator" width="6px" height="12px">
             </div>
-    
-            </div>
         </section>
 
+        <!--Setup the contact form section -->
+        
+        <footer class="footer-part">
+            <section>
+                <h3 class="contact-me">Contact me</h3>
+                <p class="call-me">If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it </p>
 
+                <form action="https://formspree.io/f/xqkwvblb" class="contact-form" method="post">
+                    <ul>
+                        <li>
+                        <input type="text" id="name" name="user_name" placeholder="Name" class="nem hint" required>
+                        </li>
+                        <li>
+                        <input type="email" id="mail" name="user_email" placeholder="E-mail" class="nem hint" required>
+                        </li>
+                        <li>
+                        <textarea id="msg" name="user_message" placeholder="Write your message here" class="nem" required></textarea>
```

Comment:
- [ ] The message input should accept up to 500 characters, please kindly fix that, as it is a requirement for this project, and a form validation rule
![abdoform](https://user-images.githubusercontent.com/65852621/140047505-270a146d-42bc-4811-a3cd-549b217abcfc.png)
.